crates/evm/fuzz/src/strategies/param.rs

```diff
@@ -132,15 +133,18 @@ pub fn fuzz_param_from_state(
         DynSolType::Address => {
             let deployed_libs = state.deployed_libs.clone();
             value()
-                .prop_filter_map("filter address fuzzed from state", move |value| {
-                    let fuzzed_addr = Address::from_word(value);
-                    // Do not use addresses of deployed libraries as fuzz input.
-                    // See <https://github.com/foundry-rs/foundry/issues/8639>.
-                    if !deployed_libs.contains(&fuzzed_addr) {
-                        Some(DynSolValue::Address(fuzzed_addr))
-                    } else {
-                        None
+                .prop_map(move |value| {
+                    let mut fuzzed_addr = Address::from_word(value);
```

Comment:
let's early return here in case `fuzzed_addr` is not a deployed lib and avoid any additional operation